package PEvent;


message PGlobalPoint { 

	required float x = 1;
	required float y = 2;
	required float z = 3;
}

message PHit {
	required PGlobalPoint position = 1;
	required uint64 layer = 2;
	required uint64 detectorId = 3;
}

message PEvent {
  required uint64 runNumber = 1;
  required uint64 lumiSection = 2;
  required uint64 eventNumber = 3;

  repeated PHit hits = 4;

}

message PEventContainer {
  repeated PEvent events = 1;

}